frontend/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+echo "installing frontend"
+npm install
+
+echo "serving frontend"
+if [ "$DEBUG" = 1 ]
+then
+    echo "develompent running frontend"
+    npm run start
+else
+    echo "production building frontend"
+    npm install -g serve
```

Comment:
Shouldn't we build it in the dockerfile instead? If we code it like this it will build the whole frontend on startup.

Reply:
npm install serve only installs serve package, does not build the static assets. Those are taken care of in the docker file as usual

Reply:
I think it should be done in the dockerfile, because it needs otherwise like 5 minutes and only returns and error in that timeframe. I will get a lot of questions from user on "How to fix it" if we keep it that way :D

Reply:
Moved back to docker file.